Scripts/Scenes/Lobby/SceneLobby.cs

```diff
@@ -55,18 +56,13 @@ public override void _Ready()
             LobbyMaxPlayers.Text = "" + CurrentLobby.MaxPlayerCount;
         }
 
-        public override void _Input(InputEvent @event)
+        public override async void _Input(InputEvent @event)
         {
             if (Input.IsActionJustPressed("ui_cancel"))
             {
                 if (NetworkManager.IsHost)
                 {
-                    Task.Run(async () =>
-                    {
-                        await WebClient.Post("servers/remove", new Dictionary<string, string> {
-                            { "Ip", WebClient.ExternalIp }
-                        });
-                    });
+                    await WebClient.RemoveAsync();
```

Comment:
The Task.Run(() => ...) was added to prevent the await from pausing execution of code. Now that await is in effect the player will have to wait for the post request to complete before their able to back out of the scene. (`NetworkManager.GameClient.Stop();` sends a request to back out of the scene)

Perhaps `await WebClient.RemoveAsync();` should be moved after `NetworkManager.GameClient.Stop();`

`WebClient.Client.CancelPendingRequests();` and `WebClient.TimerPingMasterServer.Stop();` should be moved ahead as well.